types/dd-trace/index.d.ts

```diff
@@ -1,14 +1,13 @@
-// Type definitions for dd-trace-js 0.2
+// Type definitions for dd-trace-js 0.5
 // Project: https://github.com/DataDog/dd-trace-js
 // Definitions by: Colin Bradley <https://github.com/ColinBradley>
+//                 Eloy Durán <https://github.com/alloy>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import Tracer = require('./src/opentracing/tracer');
-import Span = require('./src/opentracing/span');
-import SpanContext = require('./src/opentracing/span_context');
+import { Tracer, Span, SpanContext } from "opentracing";
 
 declare var trace: TraceProxy;
-export = trace;
+export default trace;
```

Comment:
Why have you changed this to a named export (default)? The source doesn't do this. https://github.com/DataDog/dd-trace-js/blob/master/index.js#L9

Reply:
Great catch! This was a mistake on my end because of our usage of Babel 7 to compile our TS, which has some CommonJS/ES2015 interop stuff around exports that differ from tsc. I have reverted this change and fixed it appropriately in our codebase 👍